src/web/sitemap.rs

```diff
@@ -86,6 +86,12 @@ pub fn sitemap_handler(req: &mut Request) -> IronResult<Response> {
             target_name: row.get("target_name"),
             last_modified: row
                 .get::<_, DateTime<Utc>>("release_time")
+                // On Aug 27 2022 we added `<link rel="canonical">` to all pages,
+                // so they should all get recrawled if they haven't been since then.
+                .max(DateTime::from_utc(
```

Comment:
If you `use chrono::TimeZone` this can be
```rust
                .max(Utc.ymd(2022, 8, 28).and_hms(0, 0, 0))
```

Reply:
Good tip, thanks! Done.